Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -47,6 +43,18 @@ class SearchLoader: Loader<Cursor<Site>, SearchViewController>, FeatureFlaggable
         }
     }
 
+    fileprivate func getBookmarksAsSites(matchingSearchQuery query: String, limit: UInt, completionHandler: @escaping (([Site]) -> Void)) {
+        profile.places.searchBookmarks(query: query, limit: limit).upon { result in
+            guard let bookmarkItems = result.successValue else {
+                completionHandler([])
+                return
+            }
+
+            let sites = bookmarkItems.map({ Site(url: $0.url, title: $0.title, bookmarked: true, guid: $0.guid) })
+            completionHandler(sites)
+        }
+    }
+
     private func getHistoryAsSites(matchingSearchQuery query: String, limit: Int) -> Deferred<Maybe<Cursor<Site>>> {
```

Comment:
If this function isn't used anymore we should remove it, since it's replaced with the completion handler one